webpack/webpack.prod.js

```diff
@@ -1,17 +1,62 @@
-const merge = require('webpack-merge');
-const common = require('./webpack.common');
+const merge = require('webpack-merge')
+const common = require('./webpack.common')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+const OptimizeCssAssetsPlugin = require('optimize-css-assets-webpack-plugin')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const path = require('path')
+const CleanWebpackPlugin = require('clean-webpack-plugin')
 
 module.exports = merge(common, {
   mode: 'production',
 
+  devtool: 'source-map',
+
   output: {
     filename: '[name].[chunkhash].js',
     chunkFilename: '[name].[chunkhash].chunk.js',
-    publicPath: '/',
+    publicPath: '/'
   },
 
   module: {
     rules: [
+      {
+        test: /\.(?:sa|sc|c)ss$/,
+        use: [
+          MiniCssExtractPlugin.loader,
+          {
+            loader: 'css-loader',
+            options: {
+              importLoaders: 1,
+              modules: true,
+              localIdentName: '[path][name]__[local]--[hash:base64:5]'
+            },
+          },
+          'sass-loader'
+        ]
+      }
     ]
-  }
+  },
+
+  plugins: [
+
+    new CleanWebpackPlugin(),
+
+    new MiniCssExtractPlugin({
+      filename: 'style.[contenthash].css'
+    }),
```

Comment:
As we split outputs to chunks, should probably have `chunkFileName` option for this plugin as well:
```
new MiniCssExtractPlugin({
	filename: '[name].[hash].css',
	chunkFilename: '[id].[hash].css'
}),
```